test-cypress/integration/ttyg/agent-list.spec.js

```diff
@@ -21,36 +21,22 @@ describe('TTYG agent list', () => {
         cy.wait('@get-chat');
         cy.wait('@get-all-repositories');
         // When the ttyg page is loaded
+        // Then I expect tha agent list be closed by default.
+        TTYGViewSteps.getAgentsPanel().should('not.be.visible');
+
+        // When I click on "Manage agents"
+        TTYGViewSteps.expandAgentsSidebar();
         // Then I should see the agent list with agents filtered by the current repository
-        TTYGViewSteps.getAgentsPanel().should('be.visible');
+        TTYGViewSteps.getAgentsPanel().should('not.be.visible');
```

Comment:
ummm what? Why? Shouldn't at this step the panel to become visible?

Reply:
How did you fix it? The test still does the same validation

Reply:
I'm sorry. I was looking at the commit that sais `fix broken tests` and now I saw that the fix you mentioned in the last commit